566: enable github actions and add some issue templates (#567)
eclipse/microprofile-fault-tolerance: .github/ISSUE_TEMPLATE/30-impl-proposal.md

---
name: Implementation proposal

> about: Request or propose an implementation that solves one or more use cases.

Reviewer: "Implementation proposal" seems like an odd name given that we usually use "Implementation" to mean "something that implements this spec" whereas it sounds like this template is actually for suggestions to add or change something in the spec?

Maybe "Specification change proposal" would be clearer.

Author: done

/*
 * Copyright (c) 2020 Contributors to the Eclipse Foundation
 *
 * See the NOTICE file(s) distributed with this work for additional
 * information regarding copyright ownership.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * You may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
---
name: Specification Change Proposal
about: Request or propose an implementation that solves one or more use cases.
title: ''

> labels: 'impl. proposal :gear:'

Reviewer: ```suggestion
labels: 'spec. change :gear:'
```

Author: done

assignees: ''

---

**Description**

> <!-- Describe the implementation proposal's functionality here -->

Reviewer: ```suggestion
<!-- Describe the proposed specification change here -->
```

**Use cases**

> <!-- Link at least one use case issue ID which would be solved by this implementation proposal -->

Reviewer: ```suggestion
<!-- Link at least one use case issue ID which would be solved by this specification change -->
```

Author: ok

> * Fixes #xxxx

Reviewer: We might want to add a tickbox in here somewhere to ask if this would be a breaking change.

Author: added

/*
 * Copyright (c) 2020 Contributors to the Eclipse Foundation
 *
 * See the NOTICE file(s) distributed with this work for additional
 * information regarding copyright ownership.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * You may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
---
name: Specification Change Proposal

> about: Request or propose an implementation that solves one or more use cases.

Reviewer: ```suggestion
about: Request or propose a specification change that solves one or more use cases.
```

Author: done now.
